src/views/Layout.jsx

```diff
@@ -7,7 +7,7 @@ import { auth } from '../api/config.js';
 /**
  * TODO: The links defined in this file don't work!
  *
- * Instead of anchor element, they should use a component
+ * Instead of anchor element, they should use NavLink component
  * from `react-router-dom` to navigate to the routes
  * defined in `App.jsx`.
  */
```

Comment:
Doesn't need to be now, but in general ToDo comments can be removed once the task is complete.